nucypher/cli/actions/select.py

```diff
@@ -93,64 +108,84 @@ def select_client_account(emitter,
     if show_staking:
-        headers.append('Staking')
+        headers.append("Staking")
     if show_eth_balance:
-        headers.append('ETH')
+        headers.append("ETH")
     if show_nu_balance:
-        headers.append('NU')
+        headers.append("NU")
 
     rows = list()
     for index, account in enumerated_accounts.items():
         row = [account]
         if show_staking:
             staker = Staker(domain=network, checksum_address=account, registry=registry)
             staker.refresh_stakes()
-            is_staking = 'Yes' if bool(staker.stakes) else 'No'
+            is_staking = "Yes" if bool(staker.stakes) else "No"
             row.append(is_staking)
         if show_eth_balance:
-            ether_balance = Web3.fromWei(blockchain.client.get_balance(account), 'ether')
-            row.append(f'{ether_balance} ETH')
+            ether_balance = Web3.fromWei(
+                blockchain.client.get_balance(account), "ether"
+            )
+            row.append(f"{ether_balance} ETH")
         if show_nu_balance:
-            token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=registry)
+            token_agent = ContractAgency.get_agent(
+                NucypherTokenAgent, registry=registry
+            )
             token_balance = NU.from_units(token_agent.get_balance(account, registry))
             row.append(token_balance)
         rows.append(row)
-    emitter.echo(tabulate(rows, headers=headers, showindex='always'))
+    emitter.echo(tabulate(rows, headers=headers, showindex="always"))
 
     # Prompt the user for selection, and return
     prompt = prompt or GENERIC_SELECT_ACCOUNT
-    account_range = click.IntRange(min=0, max=len(enumerated_accounts)-1)
+    account_range = click.IntRange(min=0, max=len(enumerated_accounts) - 1)
     choice = click.prompt(prompt, type=account_range, default=default)
     chosen_account = enumerated_accounts[choice]
 
-    emitter.echo(SELECTED_ACCOUNT.format(choice=choice, chosen_account=chosen_account), color='blue')
+    emitter.echo(
+        SELECTED_ACCOUNT.format(choice=choice, chosen_account=chosen_account),
+        color="blue",
+    )
     return chosen_account
 
 
-def select_network(emitter: StdoutEmitter, message: Optional[str] = None) -> str:
+def select_network(emitter: StdoutEmitter, message: Optional[str] = None, eth: Optional[bool] = True, polygon: Optional[bool] = True) -> str:
     """Interactively select a network from nucypher networks inventory list"""
-    emitter.message(message=message or str(), color='yellow')
-    rows = [[n] for n in NetworksInventory.NETWORKS]
-    emitter.echo(tabulate(rows, showindex='always'))
-    choice = click.prompt(SELECT_NETWORK, default=0, type=click.IntRange(0, len(NetworksInventory.NETWORKS)-1))
-    network = NetworksInventory.NETWORKS[choice]
+    emitter.message(message=message or str(), color="yellow")
+    if eth and polygon:
+        network_list = NetworksInventory.NETWORKS
```

Comment:
Is both eth and polygon being true possible? What's the use case there? 

If showing both lists, does it make sense to keep them as two separate lists, and show them as two separate lists...combining them into one list looks like what in the terminal? Might be confusing to have a mixed list to then select from.

Reply:
you're right, having searched through the code, there's no instance where you'd want the full list